pages/common/valgrind.md

```diff
@@ -0,0 +1,19 @@
+# valgrind
+
+> Wrapper for a set of expert tools for profiling, optimizing and debugging programs.
+
+- Test `program` with the specified Valgrind tool. Commonly used tools include `memcheck`, `cachegrind`, `callgrind`, `massif`, `helgrind`, and `drd`:
```

Comment:
Please move the "Commonly used tools..." part to the main description, as a second line.